modules/csg/csg_gizmos.cpp

```diff
@@ -317,27 +317,16 @@ bool CSGShapeSpatialGizmoPlugin::is_selectable_when_hidden() const {
 
 void CSGShapeSpatialGizmoPlugin::redraw(EditorSpatialGizmo *p_gizmo) {
 
-	CSGShape *cs = Object::cast_to<CSGShape>(p_gizmo->get_spatial_node());
```

Comment:
I wondered about this before approving and thought it was likely fine, but now I see that this seems to be a pattern for gizmo code to get that pointer before clearing, as seen in https://github.com/godotengine/godot/pull/48050/files#diff-2e3f5a3cec90a096257b4051e2b39ae52e208ab5d9331fc226f8f5e984b0f6e0R1487-R1489

So this might have broken some functionality? CC @JFonS

Reply:
It should make no difference. The pattern is most likely the result of copy-pasting code around :)